### ISSUE TITLE
New code for BVvalueType

### ISSUE THREAD

**Issue by edvinf** [ices-tools-dev/RDBES#18]
In order to support recording of determined sex of fish, this field should have an code list for sex.

**Comment by edvinf** [comment]
I removed the "New feature request" label, as we agreed on 2020 06 24 that this could, and perhaps should be looked into before before the deadline.

As mentioned at the meeting, I have tested recording age, length, weight and sex in the BV table, and have only discovered an issue with 'sex'. Have not tested maturity.

**Comment by HenrikK-N** [comment]
The code 'Sex' is now added to 'RS_ValueType'.

**Comment by edvinf** [comment]
Thanks. I believe we also need a way to specify an appropriate code in RS_UnitOfValue. As the recording of which code lists are used for categorical variables was recently move to here from a different field on BV, I am not sure we have specified a convention for how to do this, but a string identifying the appropriate ICES vocabulary would be appropriate. In addition to a code for sex, possibly the code "Maturity" in RS_UnitOfValue needs to be changed to make clear what vocabulary it refers to.

**Comment by HenrikK-N** [comment]
RS_UnitOfValue is called RS_ValueType. Codes RS_ValueType for should be taken from the existing RDB.

**Comment by davidcurrie2001** [comment]
Can I just check that at the moment people can submit anything they like in BVvalue, irrespective of what value they use in BVvalueType?  E.g. you can say that you are submitting "Length" in BVvalueType but actually submit the value "abc"?

**Comment by HenrikK-N** [comment]
Yes there is not checks at the moment. I have just send an email with suggestions for checks of the BV fields: BVtype, BVvalue and BVvalueType.

**Comment by KirstenBirchHaakansson** [comment]
BVmethod & BVmeasurementEquipment should also link to specific BVtypes - guess the same is true when we have incorporated quality / presision for the measurements
